Game2048Base/src/main/java/models/square/Square.java

```diff
@@ -0,0 +1,46 @@
+package models.square;
+
+import javalib.worldimages.Posn;
+import javalib.worldimages.WorldImage;
+
+public abstract class Square {
+    public static int SIDE_LENGTH = 100;
+    public WorldImage image;
+    public Posn position;
+    public int value;
+
```

Comment:
👾

Reply:
Where is the white space you want me to delete?